src/jaxsim/api/model.py

```diff
@@ -2373,18 +2376,24 @@ def update_λ_H_pre(joint_index):
         return jax.lax.cond(
             jnp.any(L_H_pre_mask_for_joint),
             lambda: selected_transform,
-            lambda: kin_dyn_params.joint_model.λ_H_pre[joint_index],
+            lambda: kin_dyn_params.joint_model.λ_H_pre[joint_index + 1],
         )
 
+    # lambda_H_pre should be 1+ numb of joints dim with the zero equal to  λ_H_pre.at[0].set(jnp.eye(4))
     # Apply the update function to all joint indices
     updated_λ_H_pre = jax.vmap(update_λ_H_pre)(
-        jnp.arange(kin_dyn_params.number_of_joints() + 1)
+        jnp.arange(0, kin_dyn_params.number_of_joints())
```

Comment:
The `start` argument is 0 by default
```suggestion
        jnp.arange(kin_dyn_params.number_of_joints())
```

Reply:
thanks!